docs/content/tutorials/deploy-example-app.md

```diff
@@ -33,19 +33,18 @@ Ensure that you have deployed Prometheus, Grafana, and a tracing backend and con
 
 ## Inject the Sidecar Proxy
 
-You can use either automatic injection or manual injection to add the NGINX Service Mesh sidecar containers to your application pods.
+You can use either [automatic injection]( {{< ref "/guides/inject-sidecar-proxy.md#automatic-proxy-injection" >}} ) (enabled by default) or [manual injection]( {{< ref "/guides/inject-sidecar-proxy.md#manual-proxy-injection" >}} ) to add the NGINX Service Mesh sidecar containers to your application pods.
```

Comment:
no longer enabled by default

Reply:
This is referring to auto injection via labels. In the next section down it covers how to label the namespaces correctly.
<img width="835" alt="Screenshot 2023-03-21 at 1 19 03 PM" src="https://user-images.githubusercontent.com/50845045/226731042-68a6a6ef-2893-4ed2-8ff3-940affb6d0a5.png">

Reply:
having the `(enabled by default)` note in there makes it a little confusing, I was just suggesting removing that.

Reply:
Oh...I see. 🤦 Yeah, looked right over that.